worldCupNotifier.php

```diff
@@ -132,10 +144,32 @@ function getUrl($url)
     curl_setopt_array($ch, $options);
 
     $response = curl_exec($ch);
+
     if ($response !== false)
     {
+        $response = explode("\n\r", $response);
+
+        // retrieve etag header
+        preg_match('/ETag\: "([0-9]+)"/i', $response[0], $etagMatched);
```

Comment:
If we weren't hacking this in a script, I'd recommend GuzzleHTTP to handle this!

Reply:
I totally agree